TestRange.c

```diff
@@ -0,0 +1,18 @@
+#include <assert.h>
+#include "RangeCapture.h"
+
+int main()
+{
+  int CurrentSamples[] = {3,3,5,4,10,11,12};
```

Comment:
Test data needs to have variations. For example single value {2}, repeated values {4,4,4}, no value list, so that boundary values are tested too.

Reply:
New Test Cases Added